presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -437,6 +438,10 @@ else if (serverConfig.isCoordinator()) {
         // task execution
         jaxrsBinder(binder).bind(TaskResource.class);
         jaxrsBinder(binder).bind(ThriftTaskUpdateRequestBodyReader.class);
+        install(installModuleIf(
```

Comment:
You always want to install the filters on the server. The client may be configured differently.